mobile_rewards/src/pending_txn.rs

```diff
@@ -134,11 +144,15 @@ impl PendingTxn {
         .bind(&self.hash)
         .bind(&self.txn_bin)
         .fetch_one(executor)
-        .map_err(Error::from)
-        .await
+        .await?)
```

Comment:
functionally, what conversion is this performing for us that we "unwrap" the result and then re-wrap it in an `Ok()` on success? don't we get the same result by just returning the outcome of the method chain ending in `.await` without the `?` and still get the error thrown or else an `Ok(result)`?

Reply:
It's a different error - this is a convenient way of converting an Error type into the desired one. 
The reason I do this as opposed to mapping is that try blocks will mirror something like this, as try blocks have an implicit Ok mapping their return values.